components/LottieAnimations.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState, Component } from 'react'
+import { View, StyleSheet } from 'react-native'
+import { DangerZone } from 'expo'
+const { Lottie } = DangerZone
+
+const SOURCE = require('../assets/animations/loaderTransiscope.json')
+
+export default class LottieAnimations extends Component {
+  componentDidMount() {
```

Comment:
Can be a functional component with the usage of `useEffect` hook 😃 

> Doc: https://reactjs.org/docs/hooks-effect.html

Reply:
J'y avais pensé

Reply:
Ca devrait pas prendre trop de temps, si tu as besoin d'aide hésite pas ;-)

Reply:
je vais essayer de finir aujourd'hui. la semaine a été très chargée :) .. Je suis partie avec les useRef... y a plus qu'a faire fontionner

Reply:
@ieschalier ça y est c'est fait :D , j'ai mis un peu de temps car l'app crashé. Je pensais que ca venait de mon code, mais pas du tout. Sous android, les anim de Yannis plante. J'ai eu le meme cas avant hier sur kf. Pour l'instant j'ai mis une autre anim qui marche. Je vais voir après pkoi ke JSON de l'anim transiscope pose soucis.

Reply:
Ok quand tu as la réponse partage, ça m'intéresse 😸

Reply:
https://github.com/react-native-community/lottie-react-native/issues/462#issuecomment-482151154

Reply:
Yannis doit avoir une version Bodymovin trop récente :D ...

Reply:
moyen de faire quelque chose a propos de bodymovin a l avenir ?

Reply:
en 5.4.4 normalement on est bon :smile